centinel/views.py

```diff
@@ -15,9 +17,9 @@
 from werkzeug import secure_filename
 
 
-from centinel.models import Client, Role
-import config
+from centinel.as_info import ASInfo
```

Comment:
> I know that python has some weird importing rules (e.g. it is a syntax error to try to import a filename that contains hyphens).

I checked python reference on import and didn't find any such rules. They discourage using names beginning and/or ending with hyphens in PEP8, but that's it.

> Also, our naming convention so far has been CamelCase for filenames.

Quite the opposite, I have used files with underscores in their names [before](https://github.com/iclab/centinel-server/blob/master/list_grabber.py), and so have [you](https://github.com/iclab/centinel/blob/master/centinel/experiments/check_tls_fprs.py).